test/serve_seeds_api.ts

```diff
@@ -0,0 +1,276 @@
+import supertest from 'supertest'
+import web3 from 'web3';
+
+import { MusicPlatformType } from '../src/types/platform';
+
+describe('Seeds API', () => {
+  let app: any;
+  const Web3 = new web3();
+
+  // TODO: rather stub out the permittedAdminAddresses() to contain this address
+  // instead of having to put the known address into .env
```

Comment:
Input will be appreciated on whether there is a safe and standard way of either stubbing the .env or whether I should just go ahead and stub out the `permittedAdminAddresses` method so that we aren't dependent on `.env` configuration to make tests like this pass

Reply:
maybe we could have a seperate .env.test file and then have that override the base .env when running a test? not sure best way to do it - does something like source .env && source .env.test && ... work in the package.json ?